R/PIConfiguration.R

```diff
@@ -56,24 +57,25 @@ PIConfiguration <- R6::R6Class(
       }
     },
 
-    #' @field objectiveFunctionOptions Configures model fit assessment
-    #' options. This setting influences how error and model fit are calculated
-    #' Supported options and their allowable values are detailed in
-    #' `ospsuite.parameteridentification::ObjectiveFunctionOptions`.
+    #' @field objectiveFunctionOptions Configures model fit evaluation settings,
+    #' influencing error and fit metrics. For option details and impact on cost metrics,
+    #' see \code{\link[ospsuite.parameteridentification]{ObjectiveFunctionSpecs}} and
+    #' \code{\link[ospsuite.parameteridentification]{calculateCostMetrics}}. Defaults found in
```

Comment:
You can simplify these to `[ospsuite.parameteridentification::calculateCostMetrics()]` etc

https://stackoverflow.com/questions/25489042/linking-to-other-packages-in-documentation-in-roxygen2-in-r

Reply:
That looks more clean, thanks!